src/components/expanded-state/unique-token/UniqueTokenExpandedStateHeader.tsx

```diff
@@ -325,17 +328,24 @@ const UniqueTokenExpandedStateHeader = ({
     // @ts-expect-error ContextMenu is an untyped JS component and can't type its onPress handler properly
     ({ nativeEvent: { actionKey } }) => {
       if (actionKey === FamilyActionsEnum.viewCollection && asset.marketplaceCollectionUrl) {
-        Linking.openURL(asset.marketplaceCollectionUrl);
+        openInBrowser(asset.marketplaceCollectionUrl);
       } else if (actionKey === FamilyActionsEnum.collectionWebsite) {
         // @ts-expect-error external_link and external_url could be null or undefined?
-        Linking.openURL(asset.external_link || asset.collection.external_url);
+        openInBrowser(asset.external_link || asset.collection.external_url);
       } else if (actionKey === FamilyActionsEnum.twitter) {
-        Linking.openURL('https://twitter.com/' + asset.collection.twitter_username);
+        openInBrowser('https://twitter.com/' + asset.collection.twitter_username);
       } else if (actionKey === FamilyActionsEnum.discord && asset.collection.discord_url) {
-        Linking.openURL(asset.collection.discord_url);
+        openInBrowser(asset.collection.discord_url);
```

Comment:
same here, maybe we want to open this in discord mobile if they have it downloaded?